idaes/models_extra/power_generation/unit_models/cpu.py

```diff
@@ -0,0 +1,808 @@
+        self.water_flow_mol_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=300/len(self.component_list),
+            units=flow_units,
+            doc='Water stream: Component mole flow [mol/s]')
+        self.vent_flow_mol_comp = Var(
+            self.flowsheet().config.time,
+            self.component_list,
+            initialize=2400/len(self.component_list),
+            units=flow_units,
+            doc='Vent stream: Component mole flow [mol/s]')
+
+        # Temperature [K]
+        self.inlet_temperature = Var(self.flowsheet().config.time,
+                                     initialize=110,
+                                     units=temperature_units,
+                                     doc='Inlet temperature [K]')
+        self.pureco2_temperature = Var(self.flowsheet().config.time,
+                                       initialize=110,
+                                       units=temperature_units,
+                                       doc='PureCO2 temperature [K]')
+        self.water_temperature = Var(self.flowsheet().config.time,
+                                     initialize=110,
+                                     units=temperature_units,
+                                     doc='Water temperature [K]')
+        self.vent_temperature = Var(self.flowsheet().config.time,
+                                    initialize=110,
+                                    units=temperature_units,
+                                    doc='Vent temperature [K]')
+
+        # Pressue [Pa]
+        self.inlet_pressure = Var(self.flowsheet().config.time,
+                                  initialize=17,
+                                  units=pressure_units,
+                                  doc='Inlet pressure [Pa]')
+        self.pureco2_pressure = Var(self.flowsheet().config.time,
+                                    initialize=17,
+                                    units=pressure_units,
+                                    doc='PureCO2 pressure [Pa]')
+        self.water_pressure = Var(self.flowsheet().config.time,
+                                  initialize=17,
+                                  units=pressure_units,
+                                  doc='Water pressure [Pa]')
+        self.vent_pressure = Var(self.flowsheet().config.time,
+                                 initialize=17,
+                                 units=pressure_units,
+                                 doc='Vent pressure [Pa]')
+
+        # Heat duty and work [W]
+        self.heat_duty = Var(self.flowsheet().config.time,
+                             initialize=0,
+                             units=work_units,
+                             doc='CO2 compressor cooling duty [W]')
+
+        self.work = Var(self.flowsheet().config.time,
+                        initialize=0,
+                        units=work_units,
+                        doc='CO2 compressor work [W]')
```

Comment:
Listing the units in the docstring seems redundant with defining the units on the `Var`